src/main/java/com/api/pedidosApi/services/CategoriaService.java

```diff
@@ -33,20 +30,12 @@ public Categoria Insert(Categoria obj){
 
     public boolean DeleteCategoriaIdById(Integer id) {
```

Comment:
Nomes de métodos devem iniciar com letra minúscula 
```suggestion
    public boolean deleteCategoriaIdById(Integer id) {
```